modules/reference/pages/feature/mpOpenAPI-3.0/examples.adoc

```diff
@@ -2,6 +2,9 @@
 == Examples
 
 === Configure OpenAPI documentation for a multi-module application
+
+// This example only applies to 2.0 forward, and so the file and directory must be manually copied and applied to any new versions. Copy the directory and file and change the directory name to reflect the new feature version, eg `mpOpenAPI-3.2`.
```

Comment:
```suggestion
// This example only applies to 2.0 forward, and so the file and directory must be manually copied and applied to any new versions. Copy the directory and file and change the directory name to reflect the new feature version, for example `mpOpenAPI-3.2`.
```

Reply:
I'm not sure about this. is 'and' needed in the first sentence. looks ok without it. as well